src/mercury_engine_data_structures/formats/bmtun.py

```diff
@@ -48,3 +48,20 @@ class Bmtun(BaseResource):
     @functools.lru_cache
     def construct_class(cls, target_game: Game) -> Construct:
         return BMTUN
+
+    def _check_tunable_exists(self, class_name: str, tunable_name: str) -> None:
+        classes = self.raw.classes
+        if class_name not in classes:
+            raise KeyError(f"Unknown class name: {class_name}!")
+        if tunable_name not in classes[class_name].tunables:
+            raise KeyError(f"Unknown tunable name: {tunable_name}!")
```

Comment:
actually this would be more helpful if it said something like `Unknown tunable name for {class_name}: {tunable_name}!` (update the match argument in the test as well)